en/guide/telemetry.md

```diff
@@ -0,0 +1,158 @@
+# Telemetry
+
+The [Telemetry](../api_reference/classdronecore_1_1_telemetry.md) class provides simple methods for getting vehicle telemetry, including state and flight mode information.
+
+All the methods have both synchronous and asynchronous versions, and users can set the rate at which the vehicle provides updates for each type of information.
+
+All the methods of a particular type (synchronous, asynchronous, and set_rate methods) are used in the same way.
+
+## API Overview
+
+The `Telemetry` API provides methods to return the following types of information:
+
+* [Position](../api_reference/structdronecore_1_1_telemetry_1_1_position.md) - latitude and longitude in degrees, and altitude relative to sea level and to the takeoff altitude.
+* [Battery](../api_reference/structdronecore_1_1_telemetry_1_1_battery.md) - voltage and percentage power remaining.
+* [GroundSpeedNED](http://localhost:4000/en/api_reference/structdronecore_1_1_telemetry_1_1_ground_speed_n_e_d.md) - velocity components in NED co-ordinates.

```

Comment:
`co-ordinates` -> `coordinates`?

Reply:
I'll change. FYI, both are correct, but the second is in wider usage.